src/resources/i18n/de.json

```
@@ -178,7 +178,7 @@
     "referralfrom": "Empfehlung von",
     "referralto": "Empfehlung an",
     "newpatient": "Anwesenheitstyp",
-    "newadmittance": "Neue Aufnahme",
+    "newattendance": "Neue Aufnahme",
```

When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```